src/methods/sep12/collectSep12Fields.ts

```diff
@@ -0,0 +1,54 @@
+import { log } from "helpers/log";
```

Comment:
I also moved the SEP-12 methods into their own folder so SEP-6 and SEP-31 could both use them